docs/book/src/quick-start.md

```diff
@@ -25,19 +25,25 @@ arch=$(go env GOARCH)
 
 # download kubebuilder and extract it to tmp
 curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
+```
+
+<aside class="note">
+<h1>(Only V2+) Configuring envtest</h1>
```

Comment:
Are these steps for the configuration of envtest? It looks like the text that immediately follows is focused on setting up kubebuilder, not installing/running envtest. Perhaps you meant to link to the .envtest.md page here?

If we're going to put a link to envtest docs here, we also might want to move it to the bottom of the the document (where I think it would fit better, given that this early in the quickstart no-one has things to test yet!).

Reply:
`make docker-build` will invoke `make test`, so perhaps move this to the bottom and add a link in the `Run It On the Cluster` section.

Reply:
```suggestion
<h1>(Plugin versions > v2) Configuring envtest</h1>
```

Reply:
Hi @gabbifish, 

This steps in the installation are to set up the required binaries in the /user/local/kubebuilder path and will no longer be required with this PR for plugin versions v3+. I added the link as suggested and tried to make it better.  

HI @estroz, 

We cannot move it because this steps are part of the installation see that we are moving the binaries downloaded with the kubebuilder.